src/test/e2e/cypress/README.md

```diff
@@ -4,14 +4,15 @@ This project uses [Cypress](https://www.cypress.io/) to run UI tests for the pro
 
 ## How to run the tests
```

Comment:
I think a new section in here about how to use the visual debugger is important. Unless this is possible with maven. So we have one method of using it quickly and easily through unit tests and one way that is a little harder to run but way easier to debug visually. If youd like im happy to add this.

Reply:
Can you add this to https://github.com/OWASP/wrongsecrets/issues/1105 please :-)?